snoop/views.py

```diff
@@ -127,24 +112,89 @@ def attachment_id(n):
 
     as_eml = _as_eml(doc)
 
-    for field in ['date', 'date-created']:
-        if data.get(field):
-            data[field] = _format_date(data[field])
+    data = _get_index_data_format(data)
 
     return {
         'id': id,
         'parent_id': parent_id,
         'content': data,
-        'attachments': attachments,
         'children': children,
         'as_eml': as_eml,
     }
 
-def document(request, collection_slug, id):
-    embed = request.GET.get('embed') == 'on'
-    data = _process_document(collection_slug, id)
-    data['embed'] = embed
-    return render(request, 'document.html', data)
+def _get_index_data_format(digest_data):
+    copy_keys = {
+        'path',
+        'text',
+        'subject',
+        'date',
+        'to',
+        'from',
+        'sha1',
+        'md5',
+        'lang',
+        'date-created',
+        'message-id',
+        'in-reply-to',
+        'thread-index',
+        'references',
+        'message',
+        'filename',
+        'rev',
+        'pgp',
+        'word-count',
+    }
+
+    data = {key: digest_data.get(key) for key in copy_keys}
+    data['filetype'] = digest_data.get('type')
+    data['attachments'] = bool(digest_data.get('attachments'))
+    data['people'] = ' '.join([digest_data.get('from', '')] + digest_data.get('to', []))
+    data['ocr'] = bool(digest_data.get('ocr'))
+    data['ocrtext'] = digest_data.get('ocr')
+
+    return data
 
 def document_json(request, collection_slug, id):
-    return JsonResponse(_process_document(collection_slug, id))
+    return json_response(request, _process_document(collection_slug, id))
+
+def feed(request, collection_slug):
+    collection = get_object_or_404(models.Collection, slug=collection_slug)
+
+    ids = RawSQL("(SELECT id FROM snoop_document WHERE collection_id=%s)",
+        (collection.id,))
+    query = models.Digest.objects.filter(id__in=ids).order_by('-updated_at')
+
+    if 'lt' in request.GET:
+        try:
+            lt = parser.parse(request.GET['lt'])
+        except ValueError:
+            pass
+        else:
+            query = query.filter(updated_at__lt=lt)
+
+    page_size = settings.SNOOP_FEED_PAGE_SIZE
+    page = query[:page_size]
+
+    def dump(digest):
+        digest_data = json.loads(digest.data)
+        data = _process_document(collection_slug, digest.id, digest_data)
+        version = digest.updated_at.isoformat().replace('+00:00', 'Z')
```

Comment:
I think it's safer to leave `+00:00` in place. Ultimately, the version is an opaque string that must change if the content changes, so there's no need for the replacement.